test/integration/user_views_past_orders_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.